src/routers/filing.py

```diff
@@ -35,3 +36,11 @@ async def upload_file(
 @router.get("/{lei}/filings/{filing_id}/submissions", response_model=List[SubmissionDTO])
 async def get_submission(request: Request, lei: str, filing_id: int):
     return await repo.get_submissions(request.state.db_session, filing_id)
+
+
+@router.get("/{lei}/filings/{filing_id}/submissions/latest", response_model=SubmissionDTO)
+async def get_submission_latest(request: Request, lei: str, filing_id: int):
+    results = await repo.get_submissions(request.state.db_session, filing_id, order_by="submission_time")
```

Comment:
we might want to create a method on the repo to only retrieve 1 submission based on criteria; this shouldn't be a problem under normal circumstances, but once we introduce load testing / etc. further down the lines, thousands of results might get retrieved.

Reply:
Ok I created a new repo function called get_latest_submission which specifically filters by filing_id, sorts by submission_time desc and limits the result 1.  So the db does the work.

I removed the "order_by" flag from the get_submissions since I don't think it's needed there for now.  And in the other story it was changed to use the query_helper anyway.